configuration/concepts/concepts_base.csv

```diff
@@ -0,0 +1,15 @@
+Uuid,Void/Retire,Fully specified name:en,Fully specified name:fr,Short name:en,Short name:fr,Description:en,Description:fr,Data class,Data type,_version:1,_order:1000
```

Comment:
@IipperMangesh there is no value populated for the column `Void/Retire`; if the default value is 'false`, could you please check if it's possible not to include this column entirely?